_utilities/layout-grid.md

```diff
@@ -504,116 +357,150 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
+        <td data-title="Utility" class="tablet:maxw-card-lg flex-2">
+          <span>
+            .grid-container <br />
+            .grid-container-<code>container size</code>
+          </span>
+        </td>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-container<code>(container size)</code>
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, or widescreen
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-container <br/>
+            @include grid-container("tablet")
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:maxw-card-lg display-inline-flex flex-2">
+          <span class="text-normal">
+            .grid-row
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-row
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            -
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-row
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:maxw-card-lg display-inline-flex flex-2">
+          <span class="text-normal">
+            .grid-col <br />
+            .grid-col-<code>column count</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-col<code>(column count)</code>
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            sm, md, lg, 2px, 05, 1, 3, 4, 6
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-col <br/>
+            @include grid-col(6) <br/>
+            @include grid-col("auto")
+          </span>
+        </td>
+      </tr>
+      <tr>
+        <th scope="row" data-title="Utility" class="tablet:maxw-card-lg display-inline-flex flex-2">
+          <span class="text-normal">
+            .grid-gap <br />
+            .grid-gap-<code>gap size</code>
+          </span>
+        </th>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-gap(<a href="{{ site.baseurl }}/design-tokens/spacing-units/" class="token">units</a>)
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            auto, fill, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12
```

Comment:
**issue**: incorrect grid gap values.

```suggestion
            sm, md, lg, 2px, 05, 1, 3, 4, 6
```

Reply:
Resolved in https://designsystem.digital.gov/design-tokens/order/